packages/unified-latex-util-arguments/libs/gobble-single-argument.ts

```diff
@@ -212,3 +220,69 @@ function partialStringMatch(node: Ast.Node, token: Ast.Node): boolean {
         node.content.length > token.content.length
     );
 }
+
+function cloneStringNode(node: Ast.String, content: string): Ast.String {
+    return Object.assign({}, node, { content });
+}
+
+// Find the position of the open brace and the closing brace.
```

Comment:
Use `/** ... */` style comments for doc strings.

Reply:
Fixed.